flocker/node/agents/blockdevice.py

```diff
@@ -74,10 +74,17 @@ class DatasetStates(Names):
     ATTACHED = NamedConstant()
     # Mounted on this node
     MOUNTED = NamedConstant()
+    # Symlinked to the mountpoint created, ready to be shared
+    SYMLINKED = NamedConstant()
     # Deleted from the driver
     DELETED = NamedConstant()
 
 
+# The DatasetState that corresponds to a dataset that is manifest and ready for
```

Comment:
I realized in making this PR that prior to this change "MANIFEST" meant "MOUNTED" and after this change "MANIFEST" meant "SYMLINKED".  But I had to go through the code and manually evaluate every use of "MOUNTED" to determine if it meant "The dataset is ready for use" or "the dataset has been mounted". It was sort of painful, and separating the uses makes it significantly less for future maintainers.